qutebrowser/completion/completiondelegate.py

```diff
@@ -206,7 +206,7 @@ def _get_textdoc(self, index):
             else:
                 self._doc.setPlainText(self._opt.text)
         else:
-            self._doc.setHtml('<b>{}</b>'.format(html.escape(self._opt.text)))
+            self._doc.setHtml('{}'.format(html.escape(self._opt.text)))
```

Comment:
As you only have `{}` as the format string (without anything else in it), this is the same as doing `self._doc.setHtml(html.escape(self._opt.text))`